book-1-orientation/chapters/INSTALLATIONS.md

```diff
@@ -139,6 +156,36 @@ If you haven't already set up Visual Studio Code to format your code and autosav
 
 [SQL Server Express](https://www.microsoft.com/en-us/sql-server/sql-server-downloads). This is the database tool where all your data will be stored for the server-side course. Click the Download button under the Express option. During the setup, take the basic installation option
 
+
+## Installation Check
+Now that you have finished the installation process, please double check that you have not missed something. Check that you have the following programs by pressing your Windows key (Windows keyboard) or Command key (Apple keyboard) to open the Start menue, and begin typing in the name of the following progams to see that they are installed:
+
+```ssh
+Zoom
+Slack
+Google Chrome
+Mozilla Firefox (if you installed it)
+Visual Studio
+- check for both Visual Studio & Visual Studio Code
+Postman
+Windows Terminal
+```
+
+Finally, run the following commands in Windows Terminal to check that the following are installed or set correctly:
+
+```ssh
```

Comment:
Would you mind separating these into individual code blocks and providing example output?